keybinding.go

```diff
@@ -15,6 +27,68 @@ type keybinding struct {
 	handler  func(*Gui, *View) error
 }
 
+var (
+	ErrorNotExist = errors.New("no such keybind")
+)
+
+func Parse(input string) (interface{}, Modifier, error) {
+	if len(input) == 1 {
+		_, r, err := getKey(rune(input[0]))
+		if err != nil {
+			return nil, ModNone, err
+		}
+		return r, ModNone, nil
+	}
+
+	var modifier Modifier
+	cleaned := make([]string, 0)
+
+	tokens := strings.Split(input, "+")
+	for _, t := range tokens {
+		normalized := strings.Title(strings.ToLower(t))
+		if t == "Alt" {
+			modifier = ModAlt
+			continue
+		}
+		cleaned = append(cleaned, normalized)
+	}
+
+	key, exist := translate[strings.Join(cleaned, "")]
+	if !exist {
+		return nil, ModNone, ErrorNotExist
+	}
+
+	return key, modifier, nil
+}
+
+func ParseAll(input []string) (map[interface{}]Modifier, error) {
+	ret := make(map[interface{}]Modifier)
+	for _, i := range input {
+		k, m, err := Parse(i)
+		if err != nil {
+			return ret, err
+		}
+		ret[k] = m
+	}
+	return ret, nil
+}
+
+func MustParse(input string) (interface{}, Modifier) {
```

Comment:
I'm still not sure what the Parse function exactly does but what if the user has for example a German layout and the code is trying to parse `CTRL + ß`

Reply:
From what i know, that is not supported by termbox and so it will be not be supported by gocui with this function (not saying it never will, just not with this version)

As for the what-if: Whenever the user wants to parse a keybinding that does not exists in termbox, the Parse function will throw and error, the MustParse will panic

Reply:
Due to that termbox doesn't support i think this is not that big of an issue.